app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -48,6 +46,24 @@ def submission_status
 
     private
 
+    def submit_4142(form_content, claim)
+      if form_content['form4142'].present?
+        translated_form = EVSS::DisabilityCompensationForm::Form4142.new(@current_user, form_content).translate
+        CentralMail::SubmitForm4142Job.perform_async(
+          @current_user.uuid, translated_form, claim.id, claim.created_at
+        )
+      end
+    end
+
+    def log_claim(claim)
```

Comment:
Could this be two methods grouped by use case? There's what happens in the failure case:
```ruby
StatsD.increment("#{stats_key}.failure")
raise Common::Exceptions::ValidationErrors, claim
```
and the success case:
```ruby
StatsD.increment("#{stats_key}.success")
Rails.logger.info "ClaimID=#{claim.confirmation_number} Form=#{claim.class::FORM}"
```
then `unless claim.save` can be used as a guard clause on the failure method